examples/.interop/next-prisma-starter-migration/src/pages/index.tsx

```diff
@@ -0,0 +1,125 @@
+import { trpc, proxy } from '../utils/trpc';
+import { NextPageWithLayout } from './_app';
+import Link from 'next/link';
+
+const IndexPage: NextPageWithLayout = () => {
+  const utils = trpc.useContext();
+  const postsQuery = trpc.useQuery(['post.all']);
+  const addPost = trpc.useMutation('post.add', {
+    async onSuccess() {
+      // refetches posts after a post is added
+      await utils.invalidateQueries(['post.all']);
+    },
+  });
+
+  // @ts-expect-error - should not be possible to call old procedures using proxy
+  const { data: yay } = proxy.healthz.useQuery();
+  //            ^?
```

Comment:
this should error but it doesn't.

no need to make the proxy object include old procs?